src/screenshot_gui.cpp

```diff
@@ -72,3 +72,18 @@ void ShowScreenshotWindow()
 	CloseWindowById(WC_SCREENSHOT, 0);
 	new ScreenshotWindow(&_screenshot_window_desc);
 }
+
+void SetScreenshotWindowHidden(bool hidden)
+{
+	ScreenshotWindow* scw = (ScreenshotWindow*)FindWindowById(WC_SCREENSHOT, 0);
+	if (scw != nullptr) {
+		if (hidden) {
+			scw->SetDirty();
+			SetBit(scw->left, 30);
```

Comment:
What is 30 here? Do we have a define/constant for it that can be used instead?

Reply:
30 is to make `left` very big so the window is moved out of the main screen.
I think it doesn't need to be a flag, but at least a comment to explain why 30.

Reply:
Maybe it should be something like "2 * _screen.width". Setting bit 30 is pretty arbitrary.

Reply:
I suppose then the issue becomes remembering the previous location of the window, when it's time to un-hide it. I've added a comment to explain it for now; let me know what you think.